Add filtering option for excluding specific configuration name (#25)
iurysza/module-graph: README.md

    - `NONE`: No text added. (Default.)
    - `CONFIGURATION`: The name of the configuration which the dependency belongs to (e.g. "
      implementation", "compileOnly", "jsMain").

> - **excludeConfigurationNames**:

Reviewer: Please add an example of usage in the code blocks above.  Like [here](https://github.com/iurysza/module-graph/blob/5f3117bc52144ba1542155eab3805a5b9d5ee729/README.md?plain=1#L68)

Author: added https://github.com/iurysza/module-graph/pull/25/commits/2587876249e8e69e50a4ae7b586605d790a79ae9

>   - List of configuration name which should be ignored. e.g. "implementation", "testImplementation". Default is emptyList().

Reviewer: ```suggestion
  - List of configurations to be ignored. e.g. "implementation", "testImplementation". Default is emptyList().
```

Author: As https://github.com/iurysza/module-graph/pull/25#discussion_r1540970073, https://github.com/iurysza/module-graph/pull/25#discussion_r1540969395 says 
Maybe it should be "List of configuration names to be ignored" ??

Author: fixed https://github.com/iurysza/module-graph/commit/73b5e936e23403336309bc043e887c3c0bd3aa7d
